src/dune_rules/opam_create.ml

```diff
@@ -227,6 +230,10 @@ let opam_fields project (package : Package.t) =
   let list_fields =
     [ ("maintainer", Package.Info.maintainers info)
     ; ("authors", Package.Info.authors info)
+    ; ( "license"
+      , match Package.Info.license info with
+        | None | Some [ _ ] -> None
+        | Some l -> Some l )
```

Comment:
This includes `[]` and `_::_::_`. Should we reject empty license list as well?

Reply:
I'd reject the empty list since it wasn't allowed before.

Reply:
Now rejected in parsing phase.